src/pages/home/en.md

```diff
@@ -0,0 +1,20 @@
+#Welcome to Skynet Wiki!
```

Comment:
If you want to use `#` to make things bold there needs to be a space after the `#`

Reply:
Dunno why, but in my markdown editor it was showing as working. I'll fix this in pr 3 if someone would like to merge this please.